core/templates/services/keyboard-shortcut.service.spec.ts

```diff
@@ -0,0 +1,128 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for the keyboard shortcut service.
+ */
+import 'mousetrap';
+
+import { KeyboardShortcutService } from 'services/keyboard-shortcut.service';
+
+describe('KeyboardShortcutService', () => {
+  var skipButton = document.createElement('button');
+  var nextButton = document.createElement('button');
+  var continueToNextCardButton = document.createElement('button');
+  var continueButton = document.createElement('button');
+  var backButton = document.createElement('button');
+  var searchBar = document.createElement('input');
+  var categoryBar = document.createElement('select');
+
+  const keyboardShortcutService = new KeyboardShortcutService();
+
+  beforeAll(() => {
+    skipButton.setAttribute('id', 'skipToMainContentId');
+    backButton.setAttribute('id', 'backButtonId');
+    nextButton.setAttribute('class', 'protractor-test-next-button');
+    continueButton.setAttribute('class', 'protractor-test-continue-button');
+    continueToNextCardButton.setAttribute(
+      'class', 'protractor-test-continue-to-next-card-button');
+    searchBar.setAttribute(
+      'class', 'protractor-test-search-input');
+    categoryBar.setAttribute('id', 'categoryBar');
+    document.body.append(skipButton);
+    document.body.append(continueButton);
+    document.body.append(backButton);
+    document.body.append(searchBar);
+    document.body.append(categoryBar);
+  });
+
+  it('should test navigation shortcuts.', () => {
```

Comment:
Ditto my `it` string comment below here as well.

Reply:
done